web-workout-tracker/workout/package_REMOTE_87276.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "workout",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^1.1.1",
+    "auth0-js": "^9.11.3",
+    "auth0-lock": "^11.17.2",
+    "axios": "^0.19.0",
+<<<<<<< Updated upstream
```

Comment:
It shouldn't be there. Looks like not resolved merge conflict

Reply:
done!